src/Plugin/AlwaysSeekableBodyPlugin.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Http\Client\Common\Plugin;
+
+use Http\Client\Common\Plugin;
+use Http\Message\Stream\BufferedStream;
+use Http\Promise\Promise;
+use Psr\Http\Message\RequestInterface;
+use Psr\Http\Message\ResponseInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Decorate the body of the request and the response if it's not seekable by using Http\Message\Stream\BufferedStream.
```

Comment:
s/if it's not seekable/if they are not seekable/

Reply:
are you sure about that (it = the body) ? However does not matter if splitted :p

Reply:
haha, you were right, i got confused. but yes, when we split its going to be singular on both :-)